keel-api/src/main/kotlin/com/netflix/spinnaker/keel/rest/DeliveryConfigController.kt

```diff
@@ -45,5 +55,50 @@ class DeliveryConfigController(
   fun diff(@RequestBody deliveryConfig: SubmittedDeliveryConfig): List<EnvironmentDiff> =
     adHocDiffer.calculate(deliveryConfig)
 
+  @GetMapping(
+    path = ["/{name}/environment/{environment}/constraint"],
```

Comment:
Since this returns a list of items, I think the path should be `constraints`.

Reply:
Good call, updated